src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -385,6 +389,7 @@ public final void setMessageFormat(String format) {
      * Setter to specify check ID pattern to suppress.
      *
      * @param format pattern for check ID format
+     * @since 8.24
```

Comment:
Suspicious for sure, let's fix this.

Reply:
I am failing to see how you came up with the original 8.24 .
This is `messageFormat` from the code view, and the link you gave to the documentation says `messageFormat` is since 8.6 .

Reply:
It is not suspicious and is correct.
https://checkstyle.org/releasenotes_old_8-0_8-34.html#Release_8.24
> Add ID format property to SuppressWithPlainTextCommentFilter. Author: Erik Silkensen [#6954](https://github.com/checkstyle/checkstyle/issues/6954)

Reply:
@stoyanK7 If you wanted to know some history. Here is how `since` versions were generated in 2017. Anything after 2017 was manually added in PRs for the new properties.
https://github.com/checkstyle/checkstyle/issues/4475#issuecomment-310035708

Reply:
>It is not suspicious and is correct.

How can a property be older than a check? :) I am sure that there is some bug/issue/inconsistency somewhere.

I can dig through the release notes and see: 
https://github.com/checkstyle/checkstyle/blob/dd7c5750425311131091f19f16c2a9e06f2d79e3/src/xdocs/releasenotes_old_8-0_8-34.xml#L4019-L4033

Why is release notes for 8.6 in this file anyway?

Reply:
Looks like milestones correspond to since correctly:

https://github.com/checkstyle/checkstyle/issues/6954 - 8/7/2019 - 8.24

https://github.com/checkstyle/checkstyle/issues/4841 - 7/27/2017 - 8.6

We got milestones mixed up at some point.

Reply:
@stoyanK7 I think we were mistaken on version logic :)  6 < 24

Reply:
Uhh, I see, you're right. I suppose I interpreted it as `8.2.4` in my head,

Reply:
Yes, I do the same often :)  I think I see it as `8.60` and `8.24` instead of `8.06` and `8.24`